lib/AdminPages/Pages/FormEditor.php

```diff
@@ -110,6 +112,147 @@ public function render() {
+              'border_radius' => '8',
+              'border_color' => '#f78da7',
+              'padding' => '5',
+            ],
+          ],
+        ],
+        'settings' => [
+          'segments' => ['3'],
+          'on_success' => 'message',
+          'success_message' => 'Check your inbox or spam folder to confirm your subscription.',
+          'success_page' => '5',
+          'segments_selected_by' => 'admin',
+          'alignment' => 'left',
+          'place_form_bellow_all_pages' => '',
+          'place_form_bellow_all_posts' => '',
+          'place_popup_form_on_all_pages' => '1',
+          'place_popup_form_on_all_posts' => '1',
+          'popup_form_delay' => '15',
+          'place_fixed_bar_form_on_all_pages' => '',
+          'place_fixed_bar_form_on_all_posts' => '',
+          'fixed_bar_form_delay' => '15',
+          'fixed_bar_form_position' => 'top',
+          'place_slide_in_form_on_all_pages' => '',
+          'place_slide_in_form_on_all_posts' => '',
+          'slide_in_form_delay' => '15',
+          'slide_in_form_position' => 'right',
+          'border_radius' => '0',
+          'border_size' => '0',
+          'form_padding' => '20',
+          'input_padding' => '5',
+          'below_post_styles' => ['width' => ['unit' => 'percent', 'value' => '100']],
+          'slide_in_styles' => ['width' => ['unit' => 'pixel', 'value' => '560']],
+          'fixed_bar_styles' => ['width' => ['unit' => 'percent', 'value' => '100']],
+          'popup_styles' => ['width' => ['unit' => 'pixel', 'value' => '560']],
+          'other_styles' => ['width' => ['unit' => 'percent', 'value' => '100']],
+        ],
+        'styles' => '
+            /* form */.mailpoet_form {}
+            /* columns */.mailpoet_column_with_background {  padding: 10px;}/* space between columns */.mailpoet_form_column:not(:first-child) {  margin-left: 20px;}
+            /* input wrapper (label + input) */.mailpoet_paragraph {  line-height:20px;  margin-bottom: 20px;}
+            /* labels */.mailpoet_segment_label,.mailpoet_text_label,.mailpoet_textarea_label,.mailpoet_select_label,.mailpoet_radio_label,.mailpoet_checkbox_label,.mailpoet_list_label,.mailpoet_date_label {  display:block;  font-weight: normal;}
+            /* inputs */.mailpoet_text,.mailpoet_textarea,.mailpoet_select,.mailpoet_date_month,.mailpoet_date_day,.mailpoet_date_year,.mailpoet_date {  display:block;}
+            .mailpoet_text,.mailpoet_textarea {  width: 200px;}
+            .mailpoet_checkbox {}
+            .mailpoet_submit {}
+            .mailpoet_divider {}
+            .mailpoet_message {}
+            .mailpoet_form_loading {  width: 30px;  text-align: center;  line-height: normal;}
+            .mailpoet_form_loading > span {  width: 5px;  height: 5px;  background-color: #5b5b5b;}
+        ',
+      ],
+    ];
+    if (empty($templates) || !$this->flagsController->isSupported(FeaturesController::TEMPLATES_SELECTION)) {
```

Comment:
`$templates` seems to be never empty, at least at this point.

Reply:
Hmm, looking at it now (in a later commit) the flags controller check should probably be somewhere before calling `renderTemplateSelection` and the IF removed from here completely.